src/main.rs

```diff
@@ -16,8 +16,6 @@
  * along with Compiler-Bot.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#![deny(warnings)]
```

Comment:
This is present so we don't have to pass -D in invocations of `cargo <whatever>`. I would also add deny unsafe code and clippy pedantic.